New site based on gatsby. It'll allow people to signup and fill in a form

leosoto opened the pull request:
I took the simplest template I could find and adapted it to our needs. Because there
was no way that developers were not going to complain loudly if I point them to a
webpage that is an image. 

I still plan to keep things simple and just integrate netlify and intercom and some form builder

leosoto committed:
New site based on gatsby.

I took the simplest template I could find and adapted it to our needs. Because there
was no way that developers were not going to complain loudly if I point them to a
webpage that is an image. I still plan to keep things simple and just integrate
netlify and intercom and some form builder

leonelmunozlopez commented on README.md:
```diff
@@ -1 +1,24 @@
+
+## 🚀 Quick start
+
+
```
Add the dependences time like `brew install gatsby-cli` and `npm install or yarn`

leosoto replied:
Fixed on https://github.com/continuum/talentuum-web/pull/1/commits/1e208119438a6515befa18da739112d7363e751b

leosoto committed:
Add intercom to all pages via Layout

leosoto committed:
Use regular a-href instead of Link-to.

Otherwise the Intercom widget stops working when navigating. I haven't been able
to use Gatsby SSR API to inject Intercom in a better way and I guess that
re-rendering is messing up with the Intercom widget.

TL;DR: This is a work-around and we should fix this later on

leosoto committed:
Add netlify auth and mash it together with intercom.

It's not pretty, but it works.

In the future we can re-hydrate things on the client side to use proper
react code on the browser side. In the meantime it's quite hacky.

leosoto committed:
Add deps to README and clean it up.

leosoto committed:
Add netlify config

leosoto committed:
Use yarn on netlify build

leosoto committed:
Fix typo on home page

leosoto committed:
Fix title on about page

leosoto committed:
Edit headers on home page to be more clear

leosoto committed:
Tweak value proposition a bit on home page

leosoto committed:
Show intercom after login

This should increase the chances that people continue answering
questions left unaswered on a different session/browser.
